pandas/core/internals/construction.py

```diff
@@ -701,6 +701,9 @@ def _try_cast(arr, take_fast_path, dtype, copy, raise_cast_failure):
         elif not is_extension_type(subarr):
             subarr = construct_1d_ndarray_preserving_na(subarr, dtype,
                                                         copy=copy)
+    except OutOfBoundsDatetime:
```

Comment:
we don’t raise like this instead convert to object by not here this is way too specific

Reply:
Did you read the text in the PR body above? Please respond with a bit longer and less cryptic sentence. 

Let's first discuss desired semantics before code details.

Reply:
series always work; this they need to coerce to object
Index can raise but only when the type is out of founds for it and dtype is specified); meaning floats to an Index or OOB datetimes to a DTI

not sure how this is inconsistent

further you put up code so it is fair game otherwise close this and discuss in an issue. your code is simply doing the wrong thing as i pointed.

Reply:
> further you put up code so it is fair game otherwise close this and discuss in an issue. your code is simply doing the wrong thing as i pointed.

To ask exactly this question: should we catch the error and raise, or not

Reply:
Jeff, to be clear, you have all rights to be comment on the code, since I put up a PR. I was just a bit annoyed by your first response because I found it hard to interpret (sometimes full sentences can make that easier .. :)) and ignoring my questions.